TaskHouseApi/DatabaseContext/PostgresContext.cs

```diff
@@ -1,12 +1,13 @@
-﻿using System;
+using System;
 using Microsoft.EntityFrameworkCore;
 using TaskHouseApi.Model;
 
 namespace TaskHouseApi.DatabaseContext
 {
     public class PostgresContext : DbContext
     {
-        public DbSet<User> Users { set; get; }
+         public DbSet<User> Users { set; get; }
```

Comment:
backspace for proper allignment

Reply:
fixed

Reply:
Its still an issue if you look at the diff

Reply:
```suggestion
       public DbSet<User> Users { set; get; }
```